src/openmrs-esm-error-handling.test.js

```diff
@@ -0,0 +1,53 @@
+jest.mock("./openmrs-esm-error-handling");
+const { reportError } = jest.requireActual("./openmrs-esm-error-handling");
+
+jest.useFakeTimers();
+describe("error handler", () => {
+  // it("da", () => {
+  //   showToast();
+  //   expect(showToast).toHaveBeenCalled();
+  //   expect(showToast.mock.calls[0][0]).toEqual({
+  //     description: "Something went wrong"
+  //   });
+  // });
+  it("transfrom the input in valid error object if it is not already an error obejct", () => {
+    let error;
+    try {
+      reportError("error");
+      jest.runAllTimers();
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(new Error("error"));
+
+    try {
+      reportError({ error: "error" });
+      jest.runAllTimers();
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(
+      new Error('Object thrown as error: {"error":"error"}')
+    );
+
+    try {
+      reportError(null);
+      jest.runAllTimers();
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(new Error("Object thrown as error: null"));
```

Comment:
This test is asserting the incorrect result. It should be asserting that the error message is `'null' was thrown as an error`.